apps/admin-gui/src/app/vos/pages/vo-detail-page/vo-applications/vo-applications.component.html

```diff
@@ -28,6 +28,48 @@ <h1 class="page-subtitle">{{'VO_DETAIL.APPLICATION.TITLE' | translate}}</h1>
     </button>
   </span>
   <div>
+    <span [matTooltip]="!canApprove ? tooltipMessages.approve : ''">
+      <button
+        color="accent"
+        class="me-2"
+        mat-flat-button
+        (click)="onApproveClicked()"
+        [disabled]="!canApprove">
+        {{'VO_DETAIL.APPLICATION.APPLICATION_DETAIL.APPROVE' | translate}}
+      </button>
```

Comment:
We should also check authorizations for all of the buttons. Check if principal is authorized to use bulk method.